config/software/datadog-agent-integrations.rb

```diff
@@ -111,31 +111,30 @@
       end
 
       # Copy the check config to the conf directories
-      if File.exists? "#{project_dir}/#{check}/conf.yaml.example"
-        copy "#{project_dir}/#{check}/conf.yaml.example", "#{conf_directory}/#{check}.yaml.example"
+      if File.exists? "#{project_dir}/#{check}/datadog_checks/#{check}/data/conf.yaml.example"
+        copy "#{project_dir}/#{check}/datadog_checks/#{check}/data/conf.yaml.example", "#{conf_directory}/#{check}.yaml.example"
       end
       # Copy the default config, if it exists
-      if File.exists? "#{project_dir}/#{check}/conf.yaml.default"
-        copy "#{project_dir}/#{check}/conf.yaml.default", "#{conf_directory}/#{check}.yaml.default"
+      if File.exists? "#{project_dir}/#{check}/datadog_checks/#{check}/data/conf.yaml.default"
+        copy "#{project_dir}/#{check}/datadog_checks/#{check}/data/conf.yaml.default", "#{conf_directory}/#{check}.yaml.default"
       end
 
       # We don't have auto_conf on windows yet
       unless windows?
-        if File.exists? "#{project_dir}/#{check}/auto_conf.yaml"
-          copy "#{project_dir}/#{check}/auto_conf.yaml", "#{conf_directory}/auto_conf/#{check}.yaml"
+        if File.exists? "#{project_dir}/#{check}/datadog_checks/#{check}/data/auto_conf.yaml"
+          copy "#{project_dir}/#{check}/datadog_checks/#{check}/data/auto_conf.yaml", "#{conf_directory}/auto_conf/#{check}.yaml"
         end
       end
 
       File.file?("#{project_dir}/#{check}/setup.py") || next
       if windows?
-        command("#{python_bin} -m #{python_pip}\\#{check}")
+        command("#{python_bin} -m #{python_pip_no_deps}\\#{check}")
```

Comment:
ditto re:`--no-deps`?

Reply:
As discussed offline, all the deps are now in the requirements.in file in each integration but the one that gets used is the big requirement.in file in datadog_checks_base. So we don't need to install the dependencies for a single integration anymore.